src/main/resources/lang/default.lang

```diff
@@ -1953,6 +1953,19 @@ quit reasons:
 	kicked: kicked
 	timed_out: timed out
 
+# -- Transform Reasons --
+transform reasons:
+	cured: zombie cure, zombie curing, zombie cured, cured
+	drowned: entity drowned, drowned, zombie converting to drowned, zombie drown
+	frozen: entity froze, entity frozen, skeleton frozen, skeleton froze, skeleton converting to stray
+	lightning: lightning, creeper super charge, super charged creeper
+	metamorphosis: tadpole convert, tadpole converted to frog, tadpole metamorphosis, metamorphosis
+	piglin_zombified: piglin zombification, piglin zombified
+	sheared: sheared, mooshroom sheared
+	split: slime split, slime splitting, split, magma slime split, magma slime splitting
+	unknown: unknown
+	infection: infection
```

Comment:
I think most of these should be updated to reflect the change in tense in the event syntax:
(and removing sheared, just to avoid players thinking that sheep would be involved)
```suggestion
	cured: zombie cure, zombie curing, curing
	drowned: entity drowning, drowning, zombie converting to drowned, zombie drowning
	frozen: entity freezing, entity freeze, skeleton freezing, skeleton freeze, skeleton converting to stray
	lightning: lightning, creeper super charge
	metamorphosis: tadpole converting, tadpole converting to frog, tadpole metamorphosis, metamorphosis
	piglin_zombified: piglin zombification
	sheared: mooshroom shear, mooshroom shearing
	split: slime split, slime splitting, split, magma slime split, magma slime splitting
	unknown: unknown
	infection: infection, villager infection
```
Don't forget to update examples too.

This is, of course, up to interpretation. feel free to pick what you think makes sense or if you have better ideas, let me know.